adoption/src/main/java/org/dcsa/conformance/standards/adoption/party/FilterParameter.java

```diff
@@ -17,9 +13,4 @@ public enum FilterParameter {
     this.queryParamName = queryParamName;
   }
 
-  public static final Map<String, FilterParameter> byQueryParamName =
```

Comment:
Why this has been removed?

Reply:
Because it is not needed and used. Not used code should be cleaned, unless there are very good reasons not to (e.g. going to use it soon).